src/styles/product_cards.scss

```
@@ -0,0 +1,103 @@
+.section {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: $cards-width;
+  height: $cards-height;
+  background-color: #ffff;
+  margin-bottom: 50px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  &__photo {
+    display: block;
+    width: 160px;
+    height: 134px;
+  
+    margin: 30px 19px 0;
+  }
+
+  &__header {
+    margin: 45px 14px 0;
+  }
+
+  &__title {
+    line-height: 15px;
+  }
+
+  &__code {
+    position: relative;
+    top: 4px;
+  
+    font-weight: 400;
+    font-size: $card-subtitle;
+    line-height: 13px;
+  }
+
+  &__stars {
+    @include flexbox;
+    width: 91px;
+  }
+
+  &__stars-item {
+    width: 15px;
+    height: 15px;
+  }
+
+  &__feedback {
+    font-weight: 400;
+    font-size: $card-subtitle;
+  }
+
+  &__inner {
+    @include flexbox;
+    margin: 19px 14px 0;
+    align-items: center;
+  }
+
+  &__sell {
+    @include flexbox;
+    margin: 20px 14px 0;
+    align-items: baseline;
+  }
+
+  &__price {
+    font-size: 16px;
+    font-weight: 700;
+  }
+
+  &__desription {
+    font-size: $card-subtitle;
+    font-weight: 400;
+  }
+
+  &__bottom {
+    box-sizing: border-box;
+    height: 40px;
+    background-color: #00acdc;
+    color: #fff;
+    margin: 20px 14px 15px;
+    text-align: center;
+    border-radius: 5px;
+    font-size: 14px;
+    cursor: pointer;
+    font-weight: bold;
+    border: 1px solid #00acdc;
+    text-transform: uppercase;
+    line-height: 39px;
+  }
+
+  &__bottom:hover {
+    border: 1px solid #00acdc;
+    background-color: #fff;
+    color: #00acdc;
+    transition: all 0.3s ease-in;
+  }
+  &__inner &__stars &__stars-item:nth-child(-n+4) {
```

Тобі непотрібний такий селектор
 ```suggestion
  &__stars &__stars-item:nth-child(-n+4) {
```